docs/2.7.0/docs/canton/usermanual/example_monitoring_setup.rst

```diff
@@ -8,65 +8,61 @@
 Example Monitoring Setup
 ========================
 
-This section gives an example of how canton can be run inside a connected network of docker containers. We then go on to show how network activity can be monitored. See the `Glossary`_ for an explaination of the terms used and the `Monitoring Choices`_ section on the reasoning behind the example monitoring setup. 
+This section provides an example of how Canton can be run inside a connected network of Docker containers. The example also shows how you can monitor network activity. See the `Glossary`_ for an explanation of the terms and the `Monitoring Choices`_ section for the reasoning behind the example monitoring setup. 
 
 Container Setup
 ---------------
 
-Here we go through the details of how `Docker Compose <https://docs.docker.com/compose/>`_ can be configured to spin up the docker container network shown below.
-Please see the `compose` documentation for detailed information concerning the structure of the configuration files.
+Following are the details for how you can configure `Docker Compose <https://docs.docker.com/compose/>`_ to spin up the Docker container network shown in the diagram. See the `compose` documentation for detailed information concerning the structure of the configuration files.
 
-One feature of `compose` is that it allows the overall configuration to be provided across a number of files.
-Below we look at each of the configuration files in turn and then show how we can bring them all together in a running network.
+One feature of `compose` is that you can provide the overall configuration across multiple files. Each configuration file is described below, followed by information on how to bring them together in a running network.
 
 .. image:: ./images/basic-canton-setup.svg
    :align: center
    :width: 100%
-
+   :alt: A diagram showing an example Docker network setup
 
 Intended Use
 ~~~~~~~~~~~~
 
-The purpose of of this example is to demonstrate how monitoring information can be exposed from canton, aggregated and observed. It is not suitable for production use as is without alterations. In particular note the warnings below.
+This example is intended to demonstrate how you can expose, aggregate, and observe monitoring information from Canton. It is not suitable for production without alterations. Note the following warnings.
 
  .. warning::
 
-   Ports are exposed from the docker network that are not necessary to support the UI. This may be to allow low level interaction with the underlying service via a REST interface or similar. In a production system the only ports that should be exposed are those required for the operation of the system
+   Ports are exposed from the Docker network that are not necessary to support the UI. This may be to allow low-level interaction with the underlying service via a REST or similar interface. In a production system, the only ports that should be exposed are those required for the operation of the system.
```

Comment:
```suggestion
   Ports are exposed from the Docker network that are not necessary to support the UI. For example, it can allow low-level interaction with the underlying service via a REST or similar interface. In a production system, the only ports that should be exposed are those required for the operation of the system.
```

Reply:
> 

How about "the network can" rather than "it can"?